### DISCUSSION TITLE
How can I easily just download the latest release of micropython for a specific device?

### DISCUSSION THREAD

**Discussion by TheChymera** [micropython/micropython#11366] [General]
I'd like to set up a script which updates the firmware on my devices. Is there any way to just `wget` the latest release without having to manually navigate or parse the page (e.g. https://micropython.org/download/seeed_xiao_nrf52/ ) ?

**Comment by russhughes** [comment]
Cheat and use python: https://gist.github.com/russhughes/31b14f468872cc53d08087bd496ceceb

**Reply by Josverl** [reply]
It would be nice if there was a firmwares.json or similar file on the website or repo that listed the stable and nightly builds for all ports and boards.
That would make this less dependent of changes to the websites UX.

I've seen that the `tools/autobuild/build-downloads.py` script creates such a json file. But I've not been able to determine if or where that json file is available on the website.

**Comment by TheChymera** [comment]
Thank you!
